src/obb/impl/main.cljs

```diff
@@ -47,16 +52,35 @@
   [s]
   (sci/eval-string* ctx s))
 
+(def print*
+  ;; All output from osascript by default goes to stderr. To get around this
+  ;; we use the Objective-C bridge to write directly to stdout.
+  (let [import (delay (.import js/ObjC "Foundation"))]
```

Comment:
What's the reason for the delay?

Reply:
I should have added a comment here!

My goals in using `delay` here were to (a) ensure that `Foundation` only gets imported once, and (b) defer importing it until it is actually needed. This could help with start-up times if import winds up being a relatively expensive operation.